schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml

```diff
@@ -0,0 +1,57 @@
+---
+# Default ordered sequence of steps to be optionally overwritten for s390x kvm
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta: []
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_registration/skip_module_registration
+add_on_product:
+  - installation/add_on_product/skip_install_addons
+system_role:
+  - installation/system_role/accept_selected_role_text_mode
```

Comment:
I cannot access OSD at the moment but wondering if we need to overwrite then in schedule/yam/test_cases/ha_textmode.yaml the system role or we can drop that in that file.

Reply:
I will run a test to check whether there is blocker to run accept_selected_role_text_mode instead, if no I think we can remove that overwrite in ha_textmode.yaml.

Reply:
osd is back, we can't, the ha role is preselected:
https://openqa.suse.de/tests/14611192#step/select_role_text_mode/1